src/frontend/metrics/getPercentageOfNetworkAvailability.ts

```diff
@@ -0,0 +1,10 @@
+import type {Observation} from '@mapeo/schema';
+
+export default function (observations: ReadonlyArray<Observation>) {
```

Comment:
1. Could you change this to a named export? That's the code style for the rest of this repo, I learned. 
2. I could see moving this into `generateMetricsReport.ts`, as it's really only useful there. Feel free to ignore.